Lib/test/test__xxsubinterpreters.py

```diff
@@ -759,7 +759,12 @@ def test_still_running(self):
         main, = interpreters.list_all()
         interp = interpreters.create()
         with _running(interp):
-            with self.assertRaises(RuntimeError):
+            self.assertTrue(interpreters.is_running(interp),
+                msg=f"Interp {interp} should be running before destruction.")
+
+            with self.assertRaises(RuntimeError,
+                msg=f"Should not be able to destroy interp {interp} while"
```

Comment:
I wouldn't worry about wrapping the line like this.  It's one of those cases where we break PEP 8 for practicality. :smile:

Also, consider giving this line another indent, so it is not in-line with the with statement's body.

Reply:
Yeah, I think it's much cleaner without the line wrapping in this case. But, since different core devs have different opinions on how strictly we should adhere to the PEP8 line width limit in the standard library, I just got in the habit of wrapping it by default once it exceeds 79 chars.

In your opinion, what do you think the "hard limit" should be as far as line width goes? In my own code, I typically start to think about wrapping at around 80-100 characters, and have 120 as a hard limit.

Fixed.

Reply:
Just consider that a message intended for readers shouldn't itself have too long a line length, regardless of how long it is encoded in the code.  So at a certain length (e.g. 80, 72) either the user-facing message should be shortened or wrapped (with explicit `\n` characters).  If wrapped, then it would make sense to also wrap the string in the code.  Hopefully that makes sense.  Regardless, rarely should there be a need to have such a long user-facing message.

Reply:
In this case, we're at ~65 characters for the actual user message (depending on how many chars the interpreter number takes up), so I think this one is okay. Thanks for the info!